Version 2.3.10 ignores additional INCLUDE_DIR

The installation of RSQLite 2.3.10 fails when SQLite is in a custom path. Neither exporting PKG_CONFIG_PATH=/opt/sqlite/3.49.1/lib/pkgconfig nor R CMD INSTALL --configure-vars="LIB_DIR='/opt/sqlite/3.49.1/lib' INCLUDE_DIR='/opt/sqlite/3.49.1/include'" RSQLite_2.3.10.tar.gz work. The compilation stops at ext-csv.c

gcc -I"/lisc/app/R/4.2.3/lib64/R/include" -DNDEBUG -Ivendor -DRSQLITE_USE_BUNDLED_SQLITE -DSQLITE_ENABLE_RTREE -DSQLITE_ENABLE_FTS3 -DSQLITE_ENABLE_FTS3_PARENTHESIS -DSQLITE_ENABLE_FTS5 -DSQLITE_ENABLE_JSON1 -DSQLITE_ENABLE_STAT4 -DSQLITE_SOUNDEX -DSQLITE_USE_URI=1 -DSQLITE_MAX_LENGTH=2147483647 -DHAVE_USLEEP=1 -I'/lisc/app/R/4.2.3/lib64/R/library/plogr/include' -I'/lisc/app/R/4.2.3/lib64/R/library/cpp11/include' -I/usr/local/include  -fvisibility=hidden -fpic  -g -O2  -c ext-csv.c -o ext-csv.o
In file included from ext-csv.c:5:
vendor/extensions/csv.c:42:10: fatal error: sqlite3ext.h: No such file or directory
   42 | #include <sqlite3ext.h>
      |          ^~~~~~~~~~~~~~
compilation terminated.

The problem did not exist in 2.3.9.

--- a/patch/0001-Adapt-include-paths.patch
+++ b/patch/0001-Adapt-include-paths.patch
@@ -1,0 +1,39 @@
+From 816b2a1bef8e089f95800b24d54d727f8583bee0 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Kirill=20M=C3=BCller?= <kirill@cynkra.com>
+Date: Sat, 3 May 2025 17:24:24 +0200
+Subject: [PATCH] Adapt include paths
+
+---
+ src/vendor/extensions/csv.c        | 2 +-
+ src/vendor/extensions/sqlite3ext.h | 2 +-
+ 2 files changed, 2 insertions(+), 2 deletions(-)
+
+diff --git a/src/vendor/extensions/csv.c b/src/vendor/extensions/csv.c
+index d390a558..4ff0d198 100644
+--- a/src/vendor/extensions/csv.c
++++ b/src/vendor/extensions/csv.c
+@@ -39,7 +39,7 @@
+ ** Some extra debugging features (used for testing virtual tables) are available
+ ** if this module is compiled with -DSQLITE_TEST.
+ */
+-#include <sqlite3ext.h>
++#include "../extensions/sqlite3ext.h"
+ SQLITE_EXTENSION_INIT1
+ #include <string.h>
+ #include <stdlib.h>
+diff --git a/src/vendor/extensions/sqlite3ext.h b/src/vendor/extensions/sqlite3ext.h
+index ae0949ba..afa677a1 100644
+--- a/src/vendor/extensions/sqlite3ext.h
++++ b/src/vendor/extensions/sqlite3ext.h
+@@ -17,7 +17,7 @@
+ */
+ #ifndef SQLITE3EXT_H
+ #define SQLITE3EXT_H
+-#include "sqlite3.h"
++#include "../sqlite3/sqlite3.h"
+ 
+ /*
+ ** The following structure holds pointers to all of the SQLite API
+-- 
+2.48.1
+

